restaurant-billing-app/backend/src/controllers/billingController.js

```
@@ -118,30 +118,53 @@ const billingController = {
-            track: b.track,
-            clerk_initials: b.clerk_initials,
-          })),
-        });
+            party_no: party_no.toString(), // Ensure string
+            section: "G",
+            track: track,
+            clerk_initials: clerk_initials,
+            created_at: now,
+          };
+
+          provisionalBill = await BillingModel.createProvisionalBill(
```

Biome (2.3.14)

[error] Can't assign provisionalBill because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)